javascript/ql/src/Security/CWE-116/IncompleteMultiCharacterSanitization.qhelp

```diff
@@ -1,8 +1,124 @@
 <!DOCTYPE qhelp PUBLIC
-"-//Semmle//qhelp//EN"
-"qhelp.dtd">
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
 <qhelp>
 
-	<include src="IncompleteSanitization.qhelp" />
+<overview>
+<p>
+Sanitizing untrusted input is a common technique for preventing injection attacks and other security 
+vulnerabilities. Regular expressions are often used to perform this sanitization. However, when the 
+regex matches multiple consecutive characters, applying a regular expression replacement just once 
```

Comment:
Perhaps better to consistently use "regular expression" everywhere instead of switching between that and "regex"?

```suggestion
regular expression matches multiple consecutive characters, replacing it just once 
```

Reply:
(There are a few more instances below.)